snap/snapcraft.yaml

```diff
@@ -14,7 +14,7 @@ description: |
 icon: snap/local/assets/edgex-snap-icon.png
 
 # delhi is epoch 0, edinburgh epoch 1, etc.
-epoch: 1
+epoch: 2
 
```

Comment:
Would it be worth adding a comment here stating that if it's decided that edinburgh can safely upgrade to fuji, then this should be roll this epoch back to '1'?